map.go

```diff
@@ -61,6 +61,15 @@ func NewMapFromKeys(keys Set, f func(key interface{}) interface{}) Map {
 	return b.Finish()
 }
 
+// ToFrozenMap takes a map[interface{}]interface{} and returns a frozen Map from it.
+func ToFrozenMap(m map[interface{}]interface{}) Map {
```

Comment:
Follow the same naming pattern as above: `NewMapFromGoMap`.

Reply:
change added.